src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -47,7 +45,19 @@ class SyncController(
   override def receive: Receive = idle
 
   def idle: Receive = handlePeerUpdates orElse {
-    case StartFastSync =>
+
+    case StartSync =>
+      (appStateStorage.isFastSyncDone(), doFastSync) match {
+        case (false, true) =>
+          self ! StartFastSync
```

Comment:
@ntallar I changed it so it is no longer possible to start fast sync after regular sync
https://github.com/input-output-hk/etc-client/pull/90/commits/d58dc856cbc103234c4a30980d09b996404c5290